user_manual/changes/U409-023.yaml

```diff
@@ -0,0 +1,7 @@
+type: bugfix
+title: Add Long_Long_Long_Integer declaration to standard unit
```

Comment:
Can you please add backquotes around Long_Long_Long_Integer ? (for proper sphinx rendering)